spaceship/templates/crew.html

```diff
@@ -215,7 +220,10 @@ <h5 class="modal-title">Invite Crew</h5>
 
 $(".mission-cancel").click((e) => {
   if (confirm("Are you sure?")) {
-    $.post('/mission/'+$(e.target).data('mission-id')+'/cancel').done(() => {
+    $.post('/mission/'+$(e.target).data('mission-id')+'/cancel').done((json) => {
```

Comment:
mmmm we should really have some reactive elements if we want reactive elements. this is fine for now i guess. we talked about making this into a form with the link as a submit element, but meh